Code/Editor/EditorViewportSettings.cpp

```diff
@@ -109,6 +111,15 @@ namespace SandboxEditor
                         }
                     }
                 );
+
+                m_perspectiveNotifyEventHandler = registry->RegisterNotifier(
+                    [this](const AZ::SettingsRegistryInterface::NotifyEventArgs& notifyEventArgs)
+                    {
+                        if (IsPathAncestorDescendantOrEqual(CameraFovSetting, notifyEventArgs.m_jsonKeyPath))
+                        {
+                            m_perspectiveChanged.Signal();
```

Comment:
update: It makes a lot of sense to pass through the new fov as a parameter in the signal here, it would be useful to add that here so we don't have to request it again in the callback